packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -39,7 +39,12 @@ function MyFeedHeading({
       offset={[4, 8]}
       isAlertDisabled={isAlertDisabled}
       onClose={() => onUpdateAlerts({ myFeed: null })}
-      className={{ label: 'w-44', message: 'ml-4', wrapper: 'mr-auto' }}
+      className={{
+        label: 'w-44',
+        message: !sidebarRendered ? 'ml-4' : null,
+        wrapper: 'mr-auto',
+        container: !sidebarRendered ? 'left-0' : null,
+      }}
```

Comment:
I believe you can just use the `offset` property to make adjustments to the whole pointer.
`offset={[8, 8]}`

Though I think the code in the `AlertPointer` is missing this.
In the attached image, I have modified the `bottom` property to include the unused offset parameter. The other ones should also apply the unused parameters.
![image](https://user-images.githubusercontent.com/13744167/202447764-737e898c-81c7-443a-840d-3a5492baedf3.png)

Either way, the proposed solution here fixes the concern, so I'll leave it up to you if you would want to update the `AlertPointer` file.

Reply:
Ah good point, let me modify that.